roadie-rails.gemspec

```
@@ -22,11 +22,11 @@ Gem::Specification.new do |spec|
   spec.extra_rdoc_files = %w[README.md Changelog.md LICENSE.txt]
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "railties", ">= 5.1", "< 7.1"
+  spec.add_dependency "railties", ">= 5.1", "< 7.2"
```

Can we just axe the maximum version altogether so this isn't a perpetual timebomb?  Because this is now minimally maintained, if it unexpectedly breaks in a future version, that's better than it actually working but not being installable because of an artificial restriction.

```suggestion
  spec.add_dependency "railties", ">= 5.1"
```

---

For me, no problem.
I have been trying to stay in the spirit of Mange here, by restricting one version at a time, until tested and proven to be working :)